config/_default/menus/menus.en.yaml

```
@@ -4622,146 +4622,156 @@ main:
     parent: rum_session_replay
     identifier: rum_session_replay_playlists
     weight: 305
-  - name: Heatmaps
-    url: real_user_monitoring/heatmaps
+  - name: Platform
+    identifier: real_user_monitoring_platform
+    url: real_user_monitoring/platform
     parent: rum
-    identifier: rum_heatmaps
     weight: 4
```

I think this is supposed to be 5, unless you want this above Session Replay
```suggestion
    weight: 5
```